utils/docker-ce/files/dockerd.init

```diff
@@ -52,20 +52,33 @@ uciadd() {
 		exit 0
 	}
 
+	# Add docker bridge device
+	if ! uci -q get network.docker0 >/dev/null; then
+		logger -t "dockerd-init" -p notice "Adding docker default bridge device to network uci config (docker0)"
+		uci_quiet add network device
+		uci_quiet rename network.@device[-1]="docker0"
+		uci_quiet set network.docker0.type="bridge"
+		uci_quiet set network.docker0.name="docker0"
+		uci_quiet set network.docker0.bridge_empty="1"
+		uci_quiet commit network
+	fi
+
 	# Add network interface
 	if ! uci -q get network.docker >/dev/null; then
-		logger -t "dockerd-init" -p notice "Adding docker default bridge to network uci config (docker0)"
+		logger -t "dockerd-init" -p notice "Adding docker default interface to network uci config (docker)"
 		uci_quiet add network interface
 		uci_quiet rename network.@interface[-1]="docker"
 		uci_quiet set network.docker.ifname="docker0"
 		uci_quiet set network.docker.proto="static"
 		uci_quiet set network.docker.auto="0"
+		# Add docker interface to docker0 bridge device
+		uci_quiet add_list network.docker0.ifname="docker0"
```

Comment:
Would it make sense to just swap these code blocks around?
That is, we add the interface first then when we create the bridge device we add the interface to it?

Also, does this change mean we should not `uci set network.docker0.bridge_empty="1"` anymore?

Reply:
> Would it make sense to just swap these code blocks around?
> That is, we add the interface first then when we create the bridge device we add the interface to it?

Thanks for the advice. I did.

> Also, does this change mean we should not `uci set network.docker0.bridge_empty="1"` anymore?

Also done.